charts/sscs-tribunals-api/values.preview.template.yaml

```diff
@@ -73,6 +73,25 @@ java:
     DOCUMENTS_ENGLISH_DIRECTION_ISSUED: TB-SCS-GNO-ENG-directions-notice-v2.docx
     DOCUMENTS_WELSH_DIRECTION_ISSUED: TB-SCS-GNO-ENG-directions-notice-v2.docx
     BUNDLE_ENGLISH_CONFIG: sscs-ib-bundle-config.yaml
+    BUNDLE_EDITED_CONFIG: sscs-ib-bundle-edited-config.yaml    # NOTIFICATION
+    # ==ENGLISH==
```

Comment:
unsure if this is relevant but make sure any non release ones are in a separate section so they don't get confused in release